### PR TITLE
Fix Fideslang visual explorer on docs site

### PR DESCRIPTION
**Pull request by NevilleS**
### Code Changes

* [X] Regenerate CSV files
* [X] Delete unused scripts (that don't work any more!)
* [X] Cleanup a few docs along the way

### Steps to Confirm

* [X] Ran `make docs-serve` and checked docs site locally at http://localhost:8000

| Before | After |
|---|---|
| image | image |

### Pre-Merge Checklist

* [X] All CI Pipelines Succeeded
* [X] Documentation Updated
* [X] Issue Requirements are Met
* [X] Relevant Follow-Up Issues Created
* [X] Update `CHANGELOG.md`

### Description Of Changes

_Write some things here about the changes and any potential caveats_

### CODE REVIEW

File: data_files/data_categories.csv [hunk 1]
```diff
@@ -1,58 +1,58 @@
-fides_key,is_default,name,organization_fides_key,parent_key,tags,description
```
Comment (NevilleS) [inline data_files/data_categories.csv:1]: It's hard to see this here, but the *only* difference here is line endings

I think it's just the classic Windows vs UNIX line endings problem. Tale as old as time...

File: scripts/README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-source fides-venv/bin/active
-pip install -r dev-requirements.txt
+python -m venv venv
+source venv/bin/activate
+pip install .
 ```
 
-## Generating JSON/CSV versions
+## Generating JSON/CSV versions from YAML
 The core taxonomy files are in YAML format, but for convenience it's sometimes useful to have JSON or CSV equivalents.
```
Comment (ThomasLaPiana) [inline scripts/README.md:11]: technically not correct, the core taxonomy files are in Python source code and then the YAML, JSON and CSV files all get written out from there

File: scripts/README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-python -m venv fides-venv
-source fides-venv/bin/active
-pip install -r dev-requirements.txt
+python -m venv venv
+source venv/bin/activate
+pip install .
 ```
 
-## Generating JSON/CSV versions
+## Generating JSON/CSV versions from YAML
```
Comment (ThomasLaPiana) [inline scripts/README.md]:
```suggestion
## Generating JSON/CSV/YAML files from the Taxonomy
```

**Review by ThomasLaPiana — APPROVED** [review]
One nit but otherwise LGTM!

**Pull request merged**